grid.cpp

```diff
@@ -1403,20 +1404,25 @@ void mapRefinement(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid,
 
 bool adaptRefinement(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid, FsGrid<fsgrids::technical, FS_STENCIL_WIDTH> & technicalGrid, SysBoundary& sysBoundaries, Project& project, int useStatic) {
    phiprof::Timer amrTimer {"Re-refine spatial cells"};
-   calculateScaledDeltasSimple(mpiGrid);
-
    int refines {0};
    if (useStatic > -1) {
       project.forceRefinement(mpiGrid, useStatic);
    } else {
+      // Restarts don't have all the data we need, so refinement indices need to be read
```

Comment:
So I find this contradictory: The comment states that restarts do not have all the necessary data, but the if-check on line 1412 is not performed for restarts.

Reply:
Ah, that's just a confusing comment. Restarts don't have some derivatives calculated, i.e. alphas are read from the restart file and not calculated here